src/data/repositories/TrackerProgramLoggerD2Repository.ts

```diff
@@ -25,6 +30,11 @@ export class TrackerProgramLoggerD2Repository implements LoggerRepository {
         this.messageTypeId = messageTypeId;
     }
 
+    // TODO: implement batchLog method
+    batchLog(_logs: Log[]): FutureData<void> {
+        return Future.success(undefined);
+    }
+
```

Comment:
Just as a note, a simple basic implementation of a batchLog is Future.sequential of a map of the single log.

Reply:
That sounds good! When there is time to implement also the batchLog for TrackerProgramLogger, we can implement this idea in all the loggers